src/Parser/TemplateParser.cs

```diff
@@ -110,13 +110,26 @@ private void ParseHole(CaptureType type)
         {
             int start = _position;
             int position;
-            string name = ParseName(out position); 
+            string name = ParseName(out position);
             int alignment = Peek() == ',' ? ParseAlignment() : 0;
-            string format = Peek() == ':' ? ParseFormat() : null;
-            Skip('}');
+            string format;
+            if (Peek() == ':')
+            {
+                format = ParseFormat();
+            }
+            else
+            {
+                format = null;
+                //parseFormat does the skip
```

Comment:
That's not nice/intuitive. The previous 2 liners were much better IMHO.
All it takes is make sure that `ParseFormat` returns with the position on the last brace. There are various ways to achieve that, from peeking more than one char ahead or simply doing `_position--` before returning.
Doing a decrement is really nothing compared to all the work done in parsing and that makes the code much cleaner.